### PR TITLE
[ko] Translate validation-inline

### PR DESCRIPTION
**Pull request by SilNex**

### CODE REVIEW

**Review by caouecs — CHANGES_REQUESTED** [review]
Thank you, just a problem with tag

**Comment by SilNex** [comment]
sorry click miss

**Comment by SilNex** [comment]
> Thank you, just a problem with tag

can you explain the problem of tag??

File: src/ko/validation-inline.php [hunk 1]
```diff
@@ -13,110 +13,110 @@
...
-    'same'                 => 'The value of this field must match the one from :other.',
+    'not_in'               => '선택된 값은 올바르지 않습니다.',
+    'not_regex'            => '형식이 올바르지 않습니다.',
+    'numeric'              => '숫자여야 합니다.',
+    'password'             => '비밀번호가 잘못되었습니다.',
+    'present'              => '이 필드가 있어야 합니다.',
+    'regex'                => '형식이 올바르지 않습니다.',
+    'required'             => '이 필드는 필수입니다.',
+    'required_if'          => ':other이(가) :value 일 때 이 필드는 필수입니다.',
+    'required_unless'      => ':other이(가) :values에 없다면 :attribute 이 필드는 필수입니다.',
```
Comment (caouecs) [inline src/ko/validation-inline.php:101]: the problem is here ( sorry I don't validate the review )

**Review by caouecs — CHANGES_REQUESTED** [review]
Here ;)

File: src/ko/validation-inline.php (hunk 1 above)
Comment (caouecs) [reply src/ko/validation-inline.php:101]: No :attribute in validation-inline.php

Comment (SilNex) [reply src/ko/validation-inline.php]: thank you 
i'll fix it

**Comment by caouecs** [comment]
Thank you

**Pull request merged**